inventory/host_vars/openfoodnetwork.de/config.yml

```diff
@@ -26,3 +26,4 @@ unicorn_timeout: 360
 
 # Images settings
 attachment_styles: "{\"mini\":[\"48x48#\",\"jpg\"],\"small\":[\"112x112#\",\"jpg\"],\"product\":[\"240x240>\",\"\"],\"large\":[\"600x600>\",\"jpg\"]}"
+attachment_path: ":rails_root/public/spree/products/:id/:style/:basename.:extension"
```

Comment:
this is already the default here:
https://github.com/openfoodfoundation/ofn-install/blob/0d4924ca6eb2f7811023afc49c2b860c15ca281f/roles/app/defaults/main.yml#L7